packages/common/src/authorization/projectMemberAbility.test.ts

```diff
@@ -404,7 +404,7 @@ describe('Project member permissions', () => {
                             isPrivate: false,
                         }),
                     ),
-                ).toEqual(false);
+                ).toEqual(true); // Editors can manage public dashboards
```

Comment:
again editors _can_ manage public dashboards - this test was flipped but was passing because CASL rules were wrong